EBI_indexing/update_index_json.py

```diff
@@ -60,13 +61,19 @@ def get_data_from_url(url, field=None):
                 u_dates.append(datetime.strptime(get_data_from_url(u, field=field_to_get)[2:10], "%y-%m-%d"))
             info_to_update[key + "_modified"] = max(u_dates).strftime("%y-%m-%d")
 
+        # for dataset20 (GitHub commits data), fetch the commit date
+        elif key == "dataset20":
+            commits = get_data_from_url(url)
+            commit_date = commits[0]['commit']['committer']['date'][:10]
```

Comment:
In EBI file, the expected format of the date is `YY-MM--DD`

```suggestion
            commit_date = commits[0]['commit']['committer']['date'][2:10]
```